public/style.css

```diff
@@ -234,3 +236,195 @@
+  border-radius: 3px;
+  display: grid;
+  justify-items: stretch;
+  grid-template-rows: 70px 70px 70px 50px 40px; 
+  grid-template-columns: 185px 40px 45px 40px;
+}
+
+#adultGuestDropdown {
+  grid-row-start: 1;
+  grid-row-end: 2;
+  margin-left: 20px;
+  margin-right: 20px;
+}
+
+#childGuestDropdown {
+  grid-row-start: 2;
+  grid-row-end: 3;
+  margin-left: 20px;
+  margin-right: 20px;
+}
+
+#infantGuestDropdown {
+  grid-row-start: 3;
+  grid-row-end: 4;
+  margin-left: 20px;
+  margin-right: 20px;
+}
+
+#guestsDropdownFooter {
+  grid-column-start: 1;
+  grid-column-end: 5;
+  grid-row-start: 4;
+  grid-row-end: 5;
+  color: rgb(72, 72, 72);
+  font-weight: 400;
+  font-size: 14px;
+  margin-left: 20px;
+  margin-right: 20px;
+}
+
+#closeGuestDropdown {
+  grid-column-start: 3;
+  grid-column-end: 5;
+  grid-row-start: 5;
+  grid-row-end: 6;
+
+  justify-self: right;
+  margin-right: 20px;
+}
+
+#guestsCloseButton {
+  color: rgb(0, 132, 137);
+  font-size: 16px;
+  font-weight: 600;
+  cursor: pointer;
+  outline: none;
+}
+
+#guestsCloseButton:hover {
+  text-decoration: underline;
+}
+
+#guestsCloseButton:active {
+  color: rgb(0, 92, 97);
+}
+
+.guestType {
+  color: rgb(72, 72, 72);
+  font-weight: 600;
+  font-size: 16px;
+  grid-column-start: 1;
+  grid-column-end: 2;
+}
+
+.guestDetails {
+  grid-column-start: 1;
+  grid-column-end: 2;
+  font-size:14px;
+  font-weight: 400;
+  color: rgb(72, 72, 72);
+}
+
+.addButton, .minusButton {
+  width: 30px;
+  height: 30px;
+  border-style: solid;
+  border-color: rgb(0, 132, 137);
+  border-radius: 50%;
+  border-width: 1px;
+  color: rgb(0, 132, 137);
+
+  text-align: center;
+  display: inline-block;
+  vertical-align: middle;
+}
+
+.addButton {
+  grid-column-start: 4;
+  grid-column-end: 5;
+  grid-row-start: 1;
+  grid-row-end: 3;
+  font-size:15px;
+  justify-self: center;
+  align-self: center;
+  outline:none;
+}
+.addButton:active {
+  border-color: black;
+  border-style: solid;
+  border-width: 1px;
+}
+.addButton:disabled, .minusButton:disabled {
+  color: rgb(190, 218, 220);
+  border-color: rgb(190, 218, 220);
+}
+
+.minusButton {
+  grid-column-start: 2;
+  grid-column-end: 3;
+  grid-row-start: 1;
+  grid-row-end: 3;
+  font-size:25px;
+  font-weight: 200;
+  justify-self: center;
+  align-self: center;
+  outline: none;
+}
+.minusButton:active {
+  border-color: black;
+  border-style: solid;
+  border-width: 1px;
+}
+
+.guestCounter {
+  grid-column-start: 3;
+  grid-column-end: 4;
+  grid-row-start: 1;
+  grid-row-end: 3;
+  font-size: 16px;
+  font-weight: 600;
+  color: rgb(72, 72, 72);
+  justify-self: center;
+  align-self: center;
+}
+
+.typeOfGuest {
+  display: grid;
+  grid-template-rows: 25px 45px; 
+  grid-template-columns: 165px 40px 45px 40px;
+}
+
+#adultGuestDropdown {
+  grid-template-rows: 70px; 
+}
+
+#adultAdd, #adultMinus, #adultCounter {
+  grid-row-start: 1;
+  grid-row-end: 2;
+}
+
+#adultGuest {
+  margin-top: 20px;
+}
+
+/* Popup arrow */
```

Comment:
Should be deleted before pull request

Reply:
i wanted to keep that there for reference as I may need it later